pkg/controller/volume/persistentvolume/pv_controller.go

```diff
@@ -359,8 +360,19 @@ func (ctrl *PersistentVolumeController) syncUnboundClaim(claim *v1.PersistentVol
 					return err
 				}
 				return nil
+			case storagehelpers.GetPersistentVolumeClaimClass(claim) == "":
```

Comment:
I've added some codes on top of this PR in order to:
- Work (fix wrong error check, loosen validation)
- Avoid trying to add default StorageClass when "" is intentionally set by users (although, it is actually blocked by validation)
- Only allow one default StorageClass

https://github.com/kubernetes/kubernetes/compare/master...mkimuram:storage-class-default-pv-controller-v3

It would be better tighten validation a bit more, but there won't be a big user facing change.